bitcoin/src/network/constants.rs

```diff
@@ -49,19 +52,20 @@ use crate::internal_macros::user_enum;
 /// 60001 - Support `pong` message and nonce in `ping` message
 pub const PROTOCOL_VERSION: u32 = 70001;
 
-user_enum! {
-    /// The cryptocurrency to act on
-    #[derive(Copy, PartialEq, Eq, PartialOrd, Ord, Clone, Hash, Debug)]
-    pub enum Network {
-        /// Classic Bitcoin
-        Bitcoin <-> "bitcoin",
-        /// Bitcoin's testnet
-        Testnet <-> "testnet",
-        /// Bitcoin's signet
-        Signet <-> "signet",
-        /// Bitcoin's regtest
-        Regtest <-> "regtest"
-    }
+/// The cryptocurrency network to act on.
+#[derive(Copy, PartialEq, Eq, PartialOrd, Ord, Clone, Hash, Debug)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "serde", serde(crate = "actual_serde"))]
+#[cfg_attr(feature = "serde", serde(rename_all = "lowercase"))]
+pub enum Network {
+    /// Classic Bitcoin (mainnet Bitcoin network).
```

Comment:
Would it be enough to just say "Mainnet Bitcoin"?

Reply:
That's good with me, @apoelstra?